tsconfig.base.json

```diff
@@ -5,7 +5,7 @@
     "lib": ["ES5", "ES6"],
     "moduleResolution": "Node",
     "noFallthroughCasesInSwitch": true,
-    "noImplicitAny": false,
+    "noImplicitAny": true,
```

Comment:
Do we still need this? What is missing?

Reply:
This one is in the root `tsconfig.base.json`. I removed the other ones to reduce redundancy. If we want to keep this strict typing we should keep `noImplicitAny` as true.